lib/url.c

```diff
@@ -2127,11 +2118,13 @@ static CURLcode parseurlandfillconn(struct Curl_easy *data,
        * However in RFC 1738 the "/" between the host (or port) and the
        * URL-path was NOT part of the URL-path.  Any agent that followed the
        * older spec strictly, and wanted to refer to a file with an absolute
-       * path, would have included a second slash.  So if there are two
-       * slashes, swallow one.
+       * path, would have included a second slash.  POSIX requires the OS to
+       * handle leading double slashes (or triple or quadruple) so there is
+       * no need to do anything special here for this case.
+       *
+       * Swallowing the extra "/" will actually break the UNC path resolution
+       * on Windows which is described in RFC 8089, Appendix E, Section E.2.3
```

Comment:
This looks like a typo. It should be section 3.2, right? The same mistake exists in the commit message.

Reply:
Yes, my mistake.